tests/components/timer/test_init.py

```diff
@@ -875,7 +875,7 @@ async def test_restore_idle(hass: HomeAssistant) -> None:
     await entity.async_added_to_hass()
     await hass.async_block_till_done()
     assert entity.state == STATUS_IDLE
-    assert entity.extra_state_attributes[ATTR_DURATION] == "0:00:30"
+    assert entity.extra_state_attributes[ATTR_DURATION] == "0:01:00"
```

Comment:
This test case is not valid anymore, but maybe we should keep it to simulate what happens after upgrade of Home Assistant? If so, please update the docstring. If not, remove the test.

Reply:
I kept it to reflect the new behavior but yes it's not actually testing something that can proofing anything so it's removed